### ISSUE TITLE
a11y: Form fields are missing label associations

### ISSUE THREAD

**Issue by nwmac** [rancher/dashboard#12804]
a11y Issue: 58

The form fields below do not have labels explicitly associated to them. The "for" attribute is missing in labels of the form fields resulting in screen reader users unable to understand the purpose of the form field.

Login 
- Username input field
- Password input field

Change Password 
- Current Password input field
- New Password input field
- Confirm Password input field

Create API Key
- Description input field

Cluster Appearance
- Icon Text 
- Custom comment 

Pod Security Admission: Create
- Name input field
- Description input field

Repository: Create
- Name input field
- Description input field
- Index URL (available after selecting "http(s) URL to an index generated by Helm" radio button)
- Git Repo URL (available after selecting "Git repository containing Helm chart or cluster template definitions" radio button)
- OCI Repository Host URL (available after selecting "OCI Repository Experimental" radio button)
- Min Wait Time (in Seconds), Max Wait Time (in Seconds) and Max Number of Retries  spin input fields (available after selecting "OCI Repository Experimental" radio button)

Git Repo - Add Repository
- Name input field
- Description input field
- Repository URL input field
- Branch Name input field
- Service Account Name and Target Namespace input fields (which gets available on activating "Next" button)

User: Create
- Username input field
- Display name input field
- Description input field
- New password input field
- Confirm password input field

Users Retention Settings
- Inactivity period, User retention process schedule and Default last login (ms) input fields input fields (available on activating Disable user accounts after an inactivity period (duration since last login) and Delete user accounts after an inactivity period (duration since last login) checkboxes)

Import Extension Catalog
- Catalog Image Reference

Banners
- Accept button text
-  text to display

Branding
- Private Label input field 

**Example:**

![image](https://github.com/user-attachments/assets/bfdcad0d-d663-47f3-962c-d76cb18a1903)

**Remediation:**

Use ‘for’ and ‘id’ attributes to associate labels with their respective form fields. Ensure that the value of ‘for’ attribute of the label exactly matches the value of ‘id’ attribute of the form field.

**Comment by a-arias** [comment]
**TESTS** 
Rancher : v2.11-d5a1481d04357fd04de2492ee1e7d4576ccca44c-head
UI: 
master ffe0996
Fix verified 👍
Browsers:
Google Chrome: 133.0.6943.127
Firefox: 135.0.1



Login

Username input field ✅ 
Password input field ✅ 

Change Password 

Current Password input field ✅ 
New Password input field ✅ 
Confirm Password input field✅ 

Create API Key

Description input field✅ 

Cluster Appearance

Icon Text✅ 
Custom comment✅ 

Pod Security Admission: Create

Name input field✅ 
Description input field✅ 

Repository: Create

Name input field✅ 
Description input field✅ 
Index URL (available after selecting "http(s) URL to an index generated by Helm" radio button)✅ 
Git Repo URL (available after selecting "Git repository containing Helm chart or cluster template definitions" radio button)✅ 
OCI Repository Host URL (available after selecting "OCI Repository Experimental" radio button)✅ 
Min Wait Time (in Seconds), Max Wait Time (in Seconds) and Max Number of Retries spin input fields (available after selecting "OCI Repository Experimental" radio button)✅ 

Git Repo - Add Repository

Name input field✅ 
Description input field✅ 
Repository URL input field✅ 
Branch Name input field✅ 
Service Account Name and Target Namespace input fields (which gets available on activating "Next" button)✅ 

User: Create

Username input field✅ 
Display name input field✅ 
Description input field✅ 
New password input field✅ 
Confirm password input field✅ 

Users Retention Settings

Inactivity period, User retention process schedule and Default last login (ms) input fields input fields (available on activating Disable user accounts after an inactivity period (duration since last login) and Delete user accounts after an inactivity period (duration since last login) checkboxes)✅ 

Import Extension Catalog

Catalog Image Reference✅ 

Banners

Accept button text✅ 
text to display✅ 

Branding

Private Label input field✅
